presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1476,8 +1476,8 @@ public static PlanNodeStats merge(PlanNodeStats left, PlanNodeStats right)
             long planNodeOutputPositions = left.planNodeOutputPositions + right.planNodeOutputPositions;
             DataSize planNodeOutputDataSize = succinctBytes(left.planNodeOutputDataSize.toBytes() + right.planNodeOutputDataSize.toBytes());
 
-            Map<String, OperatorInputStats> operatorInputStats = mergeOperatorInputStatsMaps(left.operatorInputStats, right.operatorInputStats);
```

Comment:
shouldn't you remove `mergeOperatorInputStatsMaps` and
`mergeOperatorHashCollisionsStatsMaps`?